memecity.game/Game/Components/ScoreComponent.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _SCORE_COMPONENT_H
+#define _SCORE_COMPONENT_H
+#include <ECS.h>
+
+struct ScoreComponent : public memecity::engine::ecs::Component
```

Comment:
Is de score niet een berekende waarde?

Reply:
alright. Zou de score dan niet in `playercomponent` kunnen? zijn er andere entiteiten die een score hebben?